docs/test_plans/tp-001-devnet-full-node.md

```diff
@@ -0,0 +1,68 @@
+# Test Plan #001: Devnet Celestia Full Node Setup
+
+## Introduction
+
+For Devnet stage, Celestia Full node can have several setup options to communicate with Celestia Core. The latter is a tendermint consensus node. For further text, we will refer to acronyms
+
+- Celestia Full Node = CFN
+- Celestia Core Node = CCN
+
+Setup options for CFN can be described with this list:
+
+1. `default` flag, which embeds CCN
+2. `--core.disable` flag, which excludes CCN
+3. `--core.remote` flag, which connects to an external CCN
+
+## In-Scope
+
+Focus on CFN setup options
+
+- Integration Testing for CFN default/remote CCN setups
```

Comment:
I think testing the relationship between CCN <> CFN should focus more on communication rather than setup -- for example, making sure that CFN is getting fed *all* new blocks from CCN. The setup is not really that concerning. 

It would be nice, for example, to dockerise CCN and CFN, spin up a couple of each on a docker network, and run the entire network for 5 minutes, dump the blocks, and make sure everything adds up.

Reply:
Thank you @renaynay for the detailed proposal. I was thinking about including communication cases into this test-plan, but thought it would be overkill. Now, I see that my concern on this point can be dropped, so I'll include the point for communication checks.
 
If you don't mind, I'll still leave some space for setup cases as I see them in the future as candidates to be run on every PR and not time consuming(I do believe that without ).

Reply:
Let's have them in the notion doc. This plan was basically covering the setup stage and basic checking that CFNs can talk to each other (block proposal, consensus, extended blocks)